airflow/cli/commands/celery_command.py

```diff
@@ -183,7 +183,7 @@ def worker(args):
 def stop_worker(args):
     """Sends SIGTERM to Celery worker"""
     # Read PID from file
-    pid_file_path, _, _, _ = setup_locations(process=WORKER_PROCESS_NAME)
+    pid_file_path = args.pid if args.pid else setup_locations(process=WORKER_PROCESS_NAME)[0]
```

Comment:
```suggestion
    if args.pid:
        pid_file_path = args.pid
    else:
        pid_file_path, _, _, _ = setup_locations(process=WORKER_PROCESS_NAME)
```

Better to fully unpack the tuple; it ensures the tuple’s length and better prevents incompatible changes in `setup_locations`.